maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/data/builder/TestModelDataBuilder.java

```diff
@@ -289,7 +293,6 @@ public static ApiCrownCourtOutcome getApiCrownCourtOutcome() {
         apiCrownCourtOutcome.setOutcome(CrownCourtOutcome.CONVICTED);
         apiCrownCourtOutcome.withOutcomeType(CrownCourtOutcome.CONVICTED.getType());
         apiCrownCourtOutcome.setDescription(CrownCourtOutcome.CONVICTED.getDescription());
-        apiCrownCourtOutcome.setDateSet(SENTENCE_ORDER_DATETIME);
```

Comment:
Is there a reason why this or the `dateSet` field in the `getOutcomeDTO` method has been removed?